### PR TITLE
Add azure cloud cost connector

### PR DESCRIPTION
**Pull request by rathodmeetsatish**
## Describe your changes
Add azure cloud cost connector

## Comment Triggers

<details>
  <summary>PR Check triggers</summary>
  
- Build: `trigger build`
- Sub Category Field Check: `trigger subcategoryfieldcheck`

### CODE REVIEW

**Review by rijajoo — APPROVED** [review]

**Review by RajBaviskar — APPROVED** [review]

File: docs/data-sources/platform_connector_azure_cloud_cost.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+- `name` (String) Name of the resource.
+- `org_id` (String) Unique identifier of the Organization.
+- `project_id` (String) Unique identifier of the Project.
+
+### Read-Only
+
+- `billing_export_spec` (List of Object) Returns Billing details like StorageAccount's Name, container's Name, directory's Name, report Name and subscription Id. (see [below for nested schema](#nestedatt--billing_export_spec))
+- `description` (String) Description of the resource.
+- `features_enabled` (Set of String) Which feature to enable among BILLING, OPTIMIZATION, VISIBILITY
+- `id` (String) The ID of this resource.
```
Comment (rashminandasahoo) [inline docs/data-sources/platform_connector_azure_cloud_cost.md:36]: Id of the resource.

File: docs/data-sources/platform_connector_azure_cloud_cost.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+- `subscription_id` (String) Subsription id.
+- `tags` (Set of String) Tags to associate with the resource. Tags should be in the form `name:value`.
```
Comment (rashminandasahoo) [inline docs/data-sources/platform_connector_azure_cloud_cost.md:38]: Tags to use for the resource. Tags should be in the form `name:value`.

File: docs/resources/platform_connector_azure_cloud_cost.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+- `features_enabled` (Set of String) Which feature to enable among BILLING, OPTIMIZATION, VISIBILITY
+- `identifier` (String) Unique identifier of the resource.
+- `name` (String) Name of the resource.
+- `subscription_id` (String) Subsription id.
+- `tenant_id` (String) Tenant id.
+
+### Optional
+
+- `billing_export_spec` (Block List, Max: 1) Returns Billing details like StorageAccount's Name, container's Name, directory's Name, report Name and subscription Id. (see [below for nested schema](#nestedblock--billing_export_spec))
```
Comment (rashminandasahoo) [inline docs/resources/platform_connector_azure_cloud_cost.md:48]: Refer to the previous comments.

File: docs/resources/platform_connector_azure_cloud_cost.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+- `description` (String) Description of the resource.
+- `org_id` (String) Unique identifier of the Organization.
```
Comment (rashminandasahoo) [inline docs/resources/platform_connector_azure_cloud_cost.md:50]: Refer to the previous comments.

File: docs/resources/platform_connector_azure_cloud_cost.md [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+- `project_id` (String) Unique identifier of the Project.
```
Comment (rashminandasahoo) [inline docs/resources/platform_connector_azure_cloud_cost.md:51]: Refer to the previous comments.

File: docs/resources/platform_connector_azure_cloud_cost.md [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+- `tags` (Set of String) Tags to associate with the resource. Tags should be in the form `name:value`.
```
Comment (rashminandasahoo) [inline docs/resources/platform_connector_azure_cloud_cost.md:52]: Refer to the previous comments.

File: docs/resources/platform_connector_azure_cloud_cost.md [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+### Read-Only
+
+- `id` (String) The ID of this resource.
+
+<a id="nestedblock--billing_export_spec"></a>
+### Nested Schema for `billing_export_spec`
+
+Required:
+
+- `container_name` (String) Container Name.
```
Comment (rashminandasahoo) [inline docs/resources/platform_connector_azure_cloud_cost.md:63]: Name of the container.

File: docs/resources/platform_connector_azure_cloud_cost.md [hunk 8]
```diff
@@ -0,0 +1,76 @@
...
+- `directory_name` (String) Directory Name.
```
Comment (rashminandasahoo) [inline docs/resources/platform_connector_azure_cloud_cost.md:64]: Name of the directory.

File: docs/resources/platform_connector_azure_cloud_cost.md [hunk 9]
```diff
@@ -0,0 +1,76 @@
...
+- `report_name` (String) Report Name.
```
Comment (rashminandasahoo) [inline docs/resources/platform_connector_azure_cloud_cost.md:65]: Name of the report.

File: docs/resources/platform_connector_azure_cloud_cost.md [hunk 10]
```diff
@@ -0,0 +1,76 @@
...
+- `storage_account_name` (String) Storage Account Name.
```
Comment (rashminandasahoo) [inline docs/resources/platform_connector_azure_cloud_cost.md:66]: Name of the storage account.

**Review by rashminandasahoo — CHANGES_REQUESTED** [review]

File: docs/data-sources/platform_connector_azure_cloud_cost.md [hunk 11]
```diff
@@ -0,0 +1,52 @@
...
+```
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Optional
+
+- `identifier` (String) Unique identifier of the resource.
+- `name` (String) Name of the resource.
+- `org_id` (String) Unique identifier of the Organization.
```
[the remark this answers is not in the document]
Comment (rathodmeetsatish) [reply docs/data-sources/platform_connector_azure_cloud_cost.md:28]: you suggested o should be capital in one of earlier pr

File: docs/data-sources/platform_connector_azure_cloud_cost.md [hunk 12]
```diff
@@ -0,0 +1,52 @@
...
+- `project_id` (String) Unique identifier of the Project.
```
[the remark this answers is not in the document]
Comment (rathodmeetsatish) [reply docs/data-sources/platform_connector_azure_cloud_cost.md:29]: you suggested p should be capital in one of earlier pr

**Review by rashminandasahoo — APPROVED** [review]

**Pull request merged**